vitest.config.e2e.ts

```diff
@@ -0,0 +1,9 @@
+import { defineConfig } from 'vite'
+
+export default defineConfig({
+  test: {
+    include: ['./tests/**/*.e2e.ts'],
+    exclude: ['./old-tests'],
```

Comment:
@alvarosabu what's the `old-tests` folder? I don't seem to find it

Reply:
I had them hidden to migrate them, I just delete it is not longer needed